worlds/blasphemous/__init__.py

```diff
@@ -67,7 +67,8 @@ def get_filler_item_name(self) -> str:
 
     def generate_early(self):
         if not self.options.starting_location.randomized:
-            if self.options.starting_location == "mourning_havoc" and self.options.difficulty < 2:
+            if (self.options.starting_location == "knot_of_words" or self.options.starting_location == "rooftops" \
```

Comment:
What's this / doing here? And why is mourning havoc a seperate line? Can it be either all in one line or seperate lines all?

Reply:
Lines can't be longer than 120 characters according to the style guidelines. (`docs/style.md`) 

The slash `\` just lets you split one line into multiple.